shared-clients/src/shared_clients/request_manager_client.py

```diff
@@ -175,11 +233,14 @@ async def send_message(
                 request_type="message",
                 metadata=metadata,
                 endpoint="generic",
+                use_responses=self.use_responses,
             )
 
-            # Extract response content
-            response_data = result.get("response", {})
-            return response_data.get("content", "No response content")
+            if debug:
```

Comment:
For later discussion/PR but we need to document/agree/and refactor so that we have consistent debug/logging across all of the components. - Create this JIRA to track - https://issues.redhat.com/browse/APPENG-3545

Reply:
I can remove these if you like. I've started standardizing by creating a shared logging library in shared-models. I guess these just slipped through.